### PR TITLE
Creates KC Cloud Alerts page

### PR DESCRIPTION
**Pull request by bstuder99**

### CODE REVIEW

File: kubecost-cloud/using-the-kubecost-cloud-ui/kubecost-cloud-alerts.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Before you finalize your alert, you can select _Test Alert_, which will send a test alert across the provided webhooks/emails. This is useful for ensuring your alert has been configured correctly. If the alert was sent successfully, you can finalize your changes by selecting _Submit_. Your alert will now appear on the Alerts page, and can be tested, edited, or deleted at any time by selecting the corresponding icons in your alert's line.
+
+## Global recipients
+
+Global recipients specify a default fallback recipient for each type of message. If an alert does not define any email recipients, its messages will be sent to any emails specified in the Global Recipients email list. Likewise, if an alert does not define a webhook, its messages will be sent to the webhook, if one is present. Alerts that do define recipients will ignore the global setting for recipients of that type.
+
+You can define global recipients by selecting _Edit_ in the Global Recipients box, then selecting the desired platform type and providing a corresponding vlaue. Confirm by selecing _Save_. You can only provide one webhook per platform as a global recipient.
```
Comment (biancaburtoiu) [inline kubecost-cloud/using-the-kubecost-cloud-ui/kubecost-cloud-alerts.md:30]:
```suggestion
You can define global recipients by selecting _Edit_ in the Global Recipients box, then selecting the desired platform type and providing a corresponding value. Confirm by selecting _Save_. You can only provide one webhook per platform as a global recipient.

```

File: kubecost-cloud/using-the-kubecost-cloud-ui/kubecost-cloud-alerts.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Cluster Status Alerts
+
+Cluster Status Alerts are unique and easy-to-generate alerts which will provide notice if there as a detected error with data coming from your cluster(s) to Kubecost.
```
Comment (biancaburtoiu) [inline kubecost-cloud/using-the-kubecost-cloud-ui/kubecost-cloud-alerts.md:34]: The description is a bit broad here IMO - could use something along these lines instead? @bstuder99 @kwombach12 What do you think?

```suggestion
Cluster Status notifications, if enabled, provide notice if and when the cluster agent(s) installed have stopped reporting data to Kubecost Cloud. 
```

File: kubecost-cloud/using-the-kubecost-cloud-ui/kubecost-cloud-alerts.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+Before you finalize your alert, you can select _Test Alert_, which will send a test alert across the provided webhooks/emails. This is useful for ensuring your alert has been configured correctly. If the alert was sent successfully, you can finalize your changes by selecting _Submit_. Your alert will now appear on the Alerts page, and can be tested, edited, or deleted at any time by selecting the corresponding icons in your alert's line.
+
+## Global recipients
+
+Global recipients specify a default fallback recipient for each type of message. If an alert does not define any email recipients, its messages will be sent to any emails specified in the Global Recipients email list. Likewise, if an alert does not define a webhook, its messages will be sent to the webhook, if one is present. Alerts that do define recipients will ignore the global setting for recipients of that type.
+
+You can define global recipients by selecting _Edit_ in the Global Recipients box, then selecting the desired platform type and providing a corresponding vlaue. Confirm by selecing _Save_. You can only provide one webhook per platform as a global recipient.
+
+## Cluster Status Alerts
```
Comment (biancaburtoiu) [inline kubecost-cloud/using-the-kubecost-cloud-ui/kubecost-cloud-alerts.md:32]: Has it been decided that we will call these 'alerts' and not eg. 'notifications'? @kwombach12

**Review by biancaburtoiu — COMMENTED** [review]
I've left a couple of questions

File: kubecost-cloud/using-the-kubecost-cloud-ui/kubecost-cloud-alerts.md (hunk 3 above)
Comment (bstuder99) [reply kubecost-cloud/using-the-kubecost-cloud-ui/kubecost-cloud-alerts.md:32]: For now, Alerts is what is in the UI which I wrote it as such

File: kubecost-cloud/using-the-kubecost-cloud-ui/kubecost-cloud-alerts.md (hunk 1 above)
Comment (bstuder99) [reply kubecost-cloud/using-the-kubecost-cloud-ui/kubecost-cloud-alerts.md:30]: Haha oops, good catch

**Review by biancaburtoiu — APPROVED** [review]
LGTM

**Pull request merged**